projects/ad9082_fmca_ebz/Readme.md

```diff
@@ -0,0 +1,9 @@
+# AD9082-FMCA-EBZ HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/eval-ad9082)
+  * Parts : [MxFE™ Quad, 16-Bit, 12GSPS RFDAC and Quad, 12-Bit, 4GSPS RFADC](https://www.analog.com/ad9081)
+  * Parts : [MxFE Quad, 16-Bit, 12 GSPS RF DAC and Dual, 12-Bit, 6 GSPS RF ADC](https://www.analog.com/ad9082)
+  * Project Doc: https://wiki.analog.com/resources/eval/user-guides/ad9081_fmca_ebz/quickstart
```

Comment:
The project is common for AD9081 and AD9082, but the project doc links is in one instance 
Project Doc: https://wiki.analog.com/resources/eval/user-guides/ad9081_fmca_ebz and in another Project Doc: https://wiki.analog.com/resources/eval/user-guides/ad9081_fmca_ebz/quickstart
The HDL is HDL Doc: https://wiki.analog.com/resources/eval/user-guides/ad9081_fmca_ebz/ad9081_fmca_ebz_hdl and   HDL Doc: https://wiki.analog.com/resources/eval/user-guides/ad9081_fmca_ebz. I think it would be best to be consistent